### PR TITLE
removed annotate from locidex search parameters

### PR DESCRIPTION
**Pull request by mattheww95**
If locidex receives a fasta that has been processed with extract it no longer needs to be annotated with pyrodigal as the sample has been blasted against the database to extract relevent hits. To verify that results generated in locidex match what is tested, this is the preferred route forward. However as databases will get larger in may be more cost effective pass in a genbank file or just annotate the fasta directly, however this relies on bakta working...

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Added locidex for allele calling. See [PR 62](https://github.com/phac-nml/mikrokondo/pull/62)
 
+- Removed `--annotate` parameter from locidex search.
```
Comment (apetkau) [inline CHANGELOG.md:18]: I would just remove this from the changelog since it's already covered by the "Added locidex" statement above.

**Review by apetkau — CHANGES_REQUESTED** [review]
Looks great. Thanks. One comment.

Also, do you know why the test MLST data changed? Does removing `--annotate` from locidex search make that large of a difference?

File: CHANGELOG.md (hunk 1 above)
Comment (mattheww95) [reply CHANGELOG.md:18]: will do, I was curiouis if it should be added. myself.

Comment (mattheww95) [reply CHANGELOG.md:18]: As for if a difference is made, test data had different outputs. I will have to consult with James about the magnitude of differences. But it definately appears to be have altered the output data, the difference seems to come from how allelese are identified in the sequences. 

There is some logic I would have to get clarification on as running with extract first may resolve paralogs. 

The method which uses just `search` with the `--annotate` flag will identify alleles/openreading frames using pyrodigal which may have some down sides. But I beleive it is more efficient at a cost of accuracy

Running extract first creates a copy the fasta used to create the blast database into the working directory, then converts the input assembly into a blast database. The original database sequences are then blasted against the sample and some additional processing is done.

Comment (mattheww95) [reply CHANGELOG.md:18]: but tl;dr we will have to seek further clarification on the impacts of each setting.

**Review by apetkau — APPROVED** [review]

**Pull request merged**